templates/geosafe/analysis/list_base.html

```diff
@@ -84,8 +84,8 @@ <h1>{% trans "List of Analysis" %}</h1>
 {#                    <th>Hazard</th>#}
 {#                    <th>Aggregation</th>#}
 {#                    <th>Extent</th>#}
-                    <th>{% trans "Impact Function Name" %}</th>
-                    <th>{% trans "Result" %}</th>
+{#                    <th>{% trans "Impact Function Name" %}</th>#}
```

Comment:
I remember someone (@gubuntu or @lucernae) asked to remove the IF name because it is no longer relevant

Reply:
Thanks @myarjunar !